### DISCUSSION TITLE
What are the plans with this project.

### DISCUSSION THREAD

**Discussion by Norbert515** [fzyzcjy/flutter_convenient_test#207] [General]
Hey there, what a great project!

I'm the author of https://www.projecttest.dev/ which plans to solve testing (more specifically end-to-end testing) for good.

To this date, I implemented a test recorder that makes it easy to record end-to-end tests. The next step was going to be to build a test UI just like this.

So I was wondering, what your plans with this project are. Maybe there could be some sort of collaboration in this space?

Feel free to also reach out via email (kozsir.norbert(AT)gmail.com).

**Comment by fzyzcjy** [comment]
Hi, your project also sounds interesting!

My open-source projects (this one, and https://github.com/fzyzcjy/flutter_rust_bridge) basically have the same situation: Originally I developed it because I have the need for my own app, and after it is mature, I open-source it such that it can help other developers :)

I am looking forward to collaborate with you. You are always welcomed to make a PR to this repository and I usually reply quite quickly. 

By the way I do not use emails that often. Just chat here or in github issues, as I read github notifications frequently.

**Reply by fzyzcjy** [reply]
@Norbert515 It is also great if the two projects could be linked together somehow. For example, users can use one GUI (one app) which has both features.
